packages/apollo-server-adonis/src/adonisApollo.ts

```diff
@@ -35,7 +35,8 @@ export function graphqlAdonis(
       query,
     }).then(
       gqlResponse => {
-        response.json(gqlResponse);
+        response.type('application/json');
```

Comment:
Thanks for tweaking that, @CaptainJojo.

One last question: Do you know if the `Content-length` is set automatically, or should we set it [like we do for `express`](https://github.com/apollographql/apollo-server/blob/fb0e4332fd593bde6d03bd5b20bcf006ba7746c9/packages/apollo-server-express/src/expressApollo.ts#L47)?  If it's not set, it would be beneficial for it to be there!

Reply:
I think is set automatically, they are not option in adonis :)